packages/lib/passport-interface/src/api/requestGenericIssuanceGetAllUserPipelines.ts

```diff
@@ -0,0 +1,26 @@
+import urlJoin from "url-join";
+import { GenericIssuanceGetAllUserPipelinesResponseValue } from "../RequestTypes";
+import { APIResult } from "./apiResult";
+import { httpGetSimple } from "./makeRequest";
+
+/**
+ * Asks the server to fetch the pipeline definition corresponding to the
+ * given pipeline ID. Requires cookies, as this is part of generic issuance
+ * user authentication.
+ */
+export async function requestGenericIssuanceGetAllUserPipelines(
+  zupassServerUrl: string
+): Promise<GenericIssuanceGetAllUserPipelinesResponse> {
+  return httpGetSimple(
+    urlJoin(zupassServerUrl, `/generic-issuance/api/pipelines`),
+    async (resText) => ({
+      value: JSON.parse(resText),
```

Comment:
Probably not worth doing now, but since we have schemas for these types, we could actually verify that the parsed JSON matches the type, rather than relying on `JSON.parse()` returning an `any`.

Reply:
I'm actually going to remove this now - ZOD parsing on the frontend slows down requests significantly.